juriscraper/opinions/united_states/state/nyappdiv_3rd.py

```diff
@@ -1,13 +1,34 @@
 # Scraper for New York Appellate Divisions 3rd Dept.
 # CourtID: nyappdiv_3rd
 # Court Short Name: NY
+import re
+from datetime import date
+from typing import Any, Dict
 
-from juriscraper.opinions.united_states.state import nyappdiv_1st
+from juriscraper.opinions.united_states.state import ny
 
 
-class Site(nyappdiv_1st.Site):
+class Site(ny.Site):
+    first_opinion_date = date(2003, 9, 25)
+    days_interval = 30
+
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
-        self.division = 3
-        self.url = self.build_url()
+        self.court = "App Div, 3d Dept"
+        self._set_parameters()
+
+    def extract_from_text(self, scraped_text: str) -> Dict[str, Any]:
+        """Can we extract the docket number from the text?
+
+        :param scraped_text: The content of the document downloaded
+        :return: Metadata to be added to the case
+        """
+        dockets = re.search(
```

Comment:
you know - those two have just a simple number, and I wasn't sure that it was actually a docket number.  so I decided to avoid saving them unless I was sure what it was.  Are you sure 113301 is the docket number for one.  I havent seen docket numbers in that format very often.

Reply:
added more patterns

Reply:
I think it is a docket number. If you use the search interface with the first docket number, it will return that case 
![image](https://github.com/user-attachments/assets/16a6fc20-8534-4335-87f4-1daacf9e91bd)